client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -530,10 +531,17 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		);
 	}
 
```

Comment:
This can be ignored for the changes required. We can pass in a prop to update the SkipLabelText value.